.github/workflows/e2e-tests-ci.yml

```diff
@@ -84,25 +80,46 @@ jobs:
           npm run check
 
   smoketests:
-    runs-on: ubuntu-latest-8-cores
+    strategy:
+      matrix:
+        #
+        # Note that smoketests should be run only on ubuntu, for QA purposes.
+        # But it's useful to be able to run and debug the smoketests for different OSes.
+        # Notes:
+        # - For MacOS: works on developer machines, but uses too many resources to be able to run on Github Actions
+        # - for Windows: cannot currently run on Github Actions, since the runners do not support running linux containers, at the moment
```

Comment:
If we cannot run windows, any reason to support that in the switch-case below here: https://github.com/mattermost/mattermost/pull/24541/files#diff-e2a82436940bb888ae5397ad9ca97d8fdcc9f8e2d20c8091170b6e603073e345R86 ?

Reply:
Running this on Windows developer machines should actually be possible, since they're able to run Linux containers. It's just that we can't do it on the `windows` Github Runners (which _can't_ run linux containers).

I recently raised this to our Github contact, that we're interested in the windows runners' Linux containers support. My hope is that we're able to periodically run E2E tests on Windows and Mac as well, to ensure that devs can also do it locally from any OS as well.